### PR TITLE
Update: line, tumblr icon in Coachtopia default

### PR DESCRIPTION
**Pull request by muhammad-irtiza04**
Ticket: https://jira.tapestry.support/browse/TORO-49549

### CODE REVIEW

**Comment by muhammad-irtiza04** [comment]
@p-musi I have added line.svg and tumblr.svg in Coachtopia  Default and I have created PR as well 
Here is the link https://github.com/Tapestry-Inc/design-tokens/pull/174

File: src/brands/coachtopia/default/asset/icon.json [hunk 1]
```diff
@@ -60,6 +60,12 @@
         },
         "authenticity": {
           "value": "authenticity.svg"
+        },
+        "line": {
+          "value": "line.svg"
+        },
+        "tumblr": {
+          "value": "tumblr.svg"
```
Comment (p-musi) [inline src/brands/coachtopia/default/asset/icon.json:68]: Hi @muhammad-irtiza04,

line.svg and tumblr.svg already exist in the global scope.

global —> asset —> icon —> social

We need them under the same path in Coachtopia so they can override.  Then can we remove the entry here in this file?

Comment (muhammad-irtiza04) [reply src/brands/coachtopia/default/asset/icon.json]: Hi @p-musi 

I have made the updates according to you mentioned above.

You can review it again

**Review by p-musi — APPROVED** [review]

**Comment by p-musi** [comment]
Screen Shot 2022-11-04 at 1 01 33 PM

I ran the build locally.  This looks good to me.  Coachtopia has the new icon files, while the other brands still retain the global version.

**Pull request merged**